crates/serpen/tests/fixtures/ast_rewriting/class_name_collision/entities.py

```
@@ -0,0 +1,34 @@
+"""
+Entities module with User and Product classes.
+These will conflict with models.py when bundled.
+"""
+
+
+class User:
+    """User entity for business logic."""
+
+    def __init__(self, name: str, age: int):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)